pkg/kubelet/dockershim/docker_sandbox_test.go

```diff
@@ -208,7 +208,6 @@ func TestNetworkPluginInvocation(t *testing.T) {
 	ds, _, _ := newTestDockerService()
 	mockPlugin := newTestNetworkPlugin(t)
 	ds.network = network.NewPluginManager(mockPlugin)
-	defer mockPlugin.Finish()
```

Comment:
Is there any specific reason we are removing the deferred calls to `mockPlugin.Finish`?

Reply:
Done

earlier the author added the finish method hand-coded, but now as the mock files are auto generated we can not add the Finish method manually. 

https://github.com/kubernetes/kubernetes/blob/026fb3bf35915c9b4e2029ce162724648854a291/pkg/kubelet/dockershim/network/testing/mock_network_plugin.go#L60-L62

I have solved this problem by defining controller

Reply:
Thanks for the context! :)